web/src/main/resources/META-INF/includes/course/courseMain.xhtml

```diff
@@ -420,13 +422,14 @@
 											value="#{courseBean.in.participationDeadline}"
 											required="false"
 											label="#{courselabels.participation_deadline}"
-											pattern="#{applicationScopeBean.inputDateTimePattern}"
+											pattern="#{sessionScopeBean.inputDateTimePattern}"
 											yearRange="#{applicationScopeBean.calendarYearRange}"
 											showWeek="true"
 											navigator="true"
 											locale="#{sessionScopeBean.locale}"
-											timeZone="#{sessionScopeBean.timeZone}"
 											styleClass="ctsms-calendar-datetime">
+											<f:converter converterId="ctsms.DateTime" />
+											<f:attribute name="timeZone" value="#{applicationScopeBean.dateUserTimeZone ? sessionScopeBean.timeZone : applicationScopeBean.defaultTimeZone}" />
```

Comment:
well this is a border case. the intention was to have a jsf-side "timezone" property so it is close to the timeZone property of pf components (which does not work properly for inputs unfortunately), since it is not uniform any longer (eg. for eCRF froms see inputFieldRow.xhtml).

introducing a combined accessor would go to sessionScopedBean as it will be session dependent value in the general case after all. but then adding fields to sessionScopeBean should be avoided to minimize bloating it.

one compromise would be to introduce a facelet parameter in baseTemplate.xhtml.
i would rather prefer to do that in a separate commit, too reisky now to destroy this commit when trying to update the pullrequest with git.